Jenkinsfile

```diff
@@ -216,7 +216,7 @@ def build(toxenv, label, imageName, artifacts, artifactExcludes) {
                                 source .venv/bin/activate
                                 # This pin must be kept in sync with tox.ini
                                 pip install coverage
-                                bash <(curl -s https://codecov.io/bash) -e JOB_BASE_NAME,LABEL,TOXENV
+                                bash <(curl -s --retry 3 https://codecov.io/bash) -e JOB_BASE_NAME,LABEL,TOXENV
```

Comment:
We've significantly altered CI in the past few days to lessen Jenkins use. This should not be necessary now.

Reply:
im removing, but the reason for this is not jenkins, it's that codecov's site is not as reliable as i might like